scripts/eia/Rwanda-RAB-AddOn.R

```diff
@@ -61,7 +60,7 @@ carob_script <- function(path) {
       #previous_crop=r$crop_cultivated_all,
       fertilizer_amount= as.numeric(r$fertiliser_amount),
       fertilizer_type= r$fertiliser_type,
-      irrigation_dates= r$Irrigation_months, 
+      irrigation_number= sapply(strsplit(r$Irrigation_months, " "), length), ## in month
       irrigation_method= r$irrigation_technique,
```

Comment:
I suggest trying to standardize the values in the `irrigation_method`.

For example anything `*bucket*` can be sub for just `bucket` and try to align to current values in [here](https://github.com/reagro/terminag/blob/3cb74d2a3954f467554365441d32985fbb83b41d/values/values_method.csv#L21) (for example `flooding` -> `uncontrolled flooding`)